### PR TITLE
feat: recover plugins meta files

### PR DESCRIPTION
**Pull request by tiennguyen-sm**

### CODE REVIEW

File: Plugins/iOS/waypoint.framework.meta
Comment (long-duc-nguyen-skymavis) [inline Plugins/iOS/waypoint.framework.meta:1]: [MUST] Make sure it is the same as before: https://github.com/skymavis/waypoint-unity/pull/18/files#diff-35911141e3fd06e8506e2d016246e962c25982998718ae53b969f8fb72362109

File: Plugins/Android/waypoint.aar.meta
Comment (long-duc-nguyen-skymavis) [inline Plugins/Android/waypoint.aar.meta:1]: [MUST] Make sure it is the same as before: https://github.com/skymavis/waypoint-unity/pull/18/files#diff-e2d951b4cf83b4fff9c6aee9d8fee5210aca45b836facc40fae6eb0e742dfd93

**Review by long-duc-nguyen-skymavis — CHANGES_REQUESTED** [review]

File: Plugins/iOS/waypoint.framework.meta
Comment (tiennguyen-sm) [reply Plugins/iOS/waypoint.framework.meta:1]: Done

File: Plugins/Android/waypoint.aar.meta
Comment (tiennguyen-sm) [reply Plugins/Android/waypoint.aar.meta:1]: Done

**Review by long-duc-nguyen-skymavis — APPROVED** [review]

**Pull request merged**